README.md

```diff
@@ -8,13 +8,23 @@ Event info: https://cps-vo.org/group/ARCH/FriendlyCompetition
 
 We plan to reuse benchmarks from 2023 but feel free to propose new benchmarks or modifications to existing ones: https://github.com/verivital/ARCH-COMP2024/issues/1
 
-Specifications modified from 2023:
-
-  - Airplane
-  
-  - Double Pendulum
-
-  - Single Pendulum
+| Benchmark | Instance | Specification | Visualization | Comment |
+|-----------|----------|---------------|---------------|---------|
+| [ACC](./benchmarks/ACC) | `TODO` | default | distance over time | - | 
+| [Airplane](./benchmarks/Airplane) | `TODO` | default | `TODO` | modified | 
+| [Attitude Control](./benchmarks/Attitude-Control) | `TODO` | default | `TODO` | - | 
+| [Benchmark 9 - Tora](./benchmarks/Benchmark9-Tora) | `TODO` | default | `TODO` | - | 
+| [Benchmark 10 - Unicycle](./benchmarks/Benchmark10-Unicycle) | `TODO` | default | `TODO` | - | 
```

Comment:
I would actually suggest to rename these two benchmarks to just TORA and Unicycle.

Reply:
I understand. This was a more general comment, not necessarily to you.